Jenkinsfile_CNP

```diff
@@ -20,7 +20,7 @@ def branchesToSync = ['demo', 'ithc', 'perftest', 'develop']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-1454"  // This doesn't change frequently, but when it does, only change this value.
```

Comment:
would this line need to be removed? I know there are entries below but I imagined we would need to point to master before merging